### PR TITLE
Feature/OP-857: Edit Offline Instructions

### PR DESCRIPTION
**Pull request by gzhou1**
- Implemented Edit Offline Instructions response workflow in modal body.
- Added validation to return error message if no changes detected in edit responses workflow.
- Return confirmation header on confirmation page based on recipient of email.

### CODE REVIEW

File: app/response/utils.py [hunk 1]
```diff
@@ -348,14 +349,15 @@ def _extension_email_handler(request_id, data, page, agency_name, email_template
...
-                           page=page)
+    email_add_extension = render_template(email_template,
+                                          default_content=default_content,
+                                          content=content,
+                                          request_id=request_id,
+                                          agency_name=agency_name,
+                                          new_due_date=new_due_date,
+                                          reason=reason,
+                                          page=page)
+    return jsonify({"template": email_add_extension}), 200
```
Comment (Palisand) [inline app/response/utils.py]: one line, you know what I mean

File: app/response/views.py [hunk 2]
```diff
@@ -254,12 +256,25 @@ def response_email():
...
+
+    Ex:
+    {
+        "json_resp": {
+            "error": "No changes detected',
+            "template": HTML rendered response template,
+            "header": "The following will be emailed to all associated participants:"
+        }
+        "status_code": 200
+    }
```
Comment (Palisand) [inline app/response/views.py]: change this to something along these lines:
```python
:return: the json response and HTTP status code
Ex1:
{
    "template": HTML rendered response template,
    "header": "The following will be emailed to all associated participants:"
}
Ex2:
{
    "error": "No changes detected."
}
```

File: app/response/views.py [hunk 3]
```diff
@@ -254,12 +256,25 @@ def response_email():
...
     """
     data = flask_request.form
     request_id = data['request_id']
-    email_template = process_email_template_request(request_id, data)
-    return email_template
+    json_resp, status_code = process_email_template_request(request_id, data)
+    return json_resp, status_code
```
Comment (Palisand) [inline app/response/views.py]: I should've realized earlier, but I think you can just return process_email_template(...)

File: app/static/js/request/view-request-responses.js [hunk 4]
```diff
@@ -330,6 +336,111 @@ $(function () {
                 });
 
                 break;
+
+            case "instructions":
```
Comment (Palisand) [inline app/static/js/request/view-request-responses.js]: include a TODO before this switch "call common function, stop copying code"

File: app/response/utils.py (hunk 1 above)
Comment (gzhou1) [inline app/response/utils.py]: I know what you mean.

File: app/response/views.py (hunk 2 above)
Comment (gzhou1) [inline app/response/views.py]: Done.

File: app/static/js/request/view-request-responses.js (hunk 4 above)
Comment (gzhou1) [inline app/static/js/request/view-request-responses.js]: Got it.

File: app/response/views.py (hunk 3 above)
Comment (gzhou1) [inline app/response/views.py]: Yes this works.

**Pull request merged**